src/course/handbook/user-research.md

```diff
@@ -0,0 +1,41 @@
+
+**[Usability testing](https://designsprintkit.withgoogle.com/methodology/phase6-validate/usability-study_1)** is a Design Sprint method used to identify any usability issues and determine the user’s satisfaction with the product prototype. This involves observing the user attempting to complete a task or set of tasks while using your product. Typically, the user is presented with different scenarios and asked to think aloud as they try to complete the tasks.
+
+## Overview
+
+- You will need **four** volunteers **each** to participate.
+  - User research (2 people) on **Monday** between **2-4pm**.
+  - Usabilitytesting (2 people) on **Thursday** between **10-12pm** .
+- Each research session will last 2 hours whereby each interview will take around **20-30minutes** to conduct.
+- It will be **1-1 interviews** over zoom, or face to face wherever possible.
+- You will conduct the research in **pairs**, where one person will be taking the **lead** and asking the questions, whereas the other person will be taking **notes** on the volunteers answers.
+- You will swap roles between interviews so that each pair has the opportunity to lead and take notes.
+
+## User Research
+
+This will take place on **Monday** between **2-4pm** of design week after the [Discovery Workshop](https://docs.google.com/presentation/d/10t83KG0ZHP5jKpxdL5HX8hkQDqvf2f01qJs-NkeqjX8/edit).
+
+- We develop products with the user's needs in mind, so that the services we design must respond to the needs of the user.
+- The purpose of this research is to find out how our users perceive the problem we are aiming to solve. It gives us an opportunity to validate whether we’re on the right track or we’re designing something that wouldn’t address the users needs.
+- You be asking the particpant questions from a script prepared ahead of time and taking notes of their answers.
+- It is important that you help the user feel comfortable and ask open questions as well as giving the user space to share their own thoughts.
+- Users might not have participated in research before so there often is a tendency to tell us what we’re expecting to hear. But critical feedback is incredibly useful to us so we want to encourage our users to be open and honest.
+- Wherever possible, we want to avoid including our own biases in questions and conversations as part of user research.
+- Always be mindful of the particpants right to privacy.
+- Once the interviews are all conducted you will synthesise the feedback in your project groups during the [Definition workshop](https://docs.google.com/presentation/d/15c3DstgW4W-cFAjTV3LRNuvS9D7Ny5_UMOMYqW-Nvj0/edit).
+
+## Usability Testing
+
+[This](https://docs.google.com/presentation/d/10VxYiFBmwLevEH3V6AP74ibf_JRt51Gd-295TICrWQU/edit#slide=id.g87e8f70714_0_133) will take place on **Thursday** between **10-12pm** of design week after [Figma wireframing](https://learn.foundersandcoders.com/course/syllabus/projects/in-house-design/schedule/).
```

Comment:
It's generally a good idea to make sure link text is descriptive of where the link goes (i.e. a screen reader user might just hear "this, link" and be confused)
```suggestion
[Usability testing](https://docs.google.com/presentation/d/10VxYiFBmwLevEH3V6AP74ibf_JRt51Gd-295TICrWQU/edit#slide=id.g87e8f70714_0_133) will take place on **Thursday** between **10-12pm** of design week after [Figma wireframing](https://learn.foundersandcoders.com/course/syllabus/projects/in-house-design/schedule/).
```

Reply:
I totally forgot about screen readers! 🤦‍♀️ Thank you!